### ISSUE TITLE
"error MSB4057: The target "_CopyWebApplication" does not exist in the project" on dotnet/framework/sdk after the Visual Studio 2019 toolset update

### ISSUE THREAD

**Issue by ytyukhnin** [Microsoft/dotnet-framework-docker#268]
Hi, 

Since the recent upgrade of the _dotnet/framework/sdk_ image with the Visual Studio 2019 toolset I started to experience difficulties while building my projects using the new MSBuild 16 tools. The error I'm getting is:

```
MSBuild.exe  "C:\app\GitLabTestApp\GitLabTestApp.csproj" "/t:_CopyWebApplication;_BuiltWebOutputGroupOutput" "/m" "/nodeReuse:False" "/clp:ForceConsoleColor" "/p:RestorePackages=False" "/p:Configuration=Release" "/p:OutDir=C:%5Capp%5Cbin%5Cweb" "/p:WebProjectOutputDir=C:%5Capp%5Cbin%5Cweb/GitLabTestApp"
Microsoft (R) Build Engine version 16.0.461+g6ff56ef63c for .NET Framework
Copyright (C) Microsoft Corporation. All rights reserved.

Build started 4/12/2019 9:09:45 AM.
     1>Project "C:\app\GitLabTestApp\GitLabTestApp.csproj" on node 1 (_CopyWebApplication;_BuiltWebOutputGroupOutput target(s)).
     1>C:\app\GitLabTestApp\GitLabTestApp.csproj : error MSB4057: The target "_CopyWebApplication" does not exist in the project.
     1>Done Building Project "C:\app\GitLabTestApp\GitLabTestApp.csproj" (_CopyWebApplication;_BuiltWebOutputGroupOutput target(s)) -- FAILED.
```

This builds completely fine on the previous version of the _dotnet/framework/sdk_ with MSBuild 15.

**Comment by mthalman** [comment]
@ytyukhnin - Could you provide repro steps starting from a new project or referencing an existing public project?  Otherwise, it'd be good to have the full MSBuild log with verbosity set to diagnostic as well as a copy of your specific project.  If you don't want that exposed publicly, you can send to me directly at mthalman@microsoft.com.

**Comment by ytyukhnin** [comment]
@mthalman - Ok, I'll try to do it during this weekend, or otherwise on Monday.

**Comment by ytyukhnin** [comment]
@mthalman - So, we've figured out what is going on. The build on our CI server was executed using a latest _dotnet/framework/sdk_ image with the environment variable _VisualStudioVersion=15.0_ set. In this case the build fails with error I mentioned in the issue. I created a test app in any case you'd like to try it https://github.com/ytyukhnin/GitLabTestApp
But it's not directly related to the docker image as we've reproduced it locally as well. 
I apologize for troubling.
